packages/react-table/src/components/TableComposable/Td.tsx

```diff
@@ -47,6 +48,15 @@ export interface TdProps extends BaseCellProps, Omit<React.HTMLProps<HTMLTableDa
   noPadding?: boolean;
   /** Applies pf-c-table__action to td */
   isActionCell?: boolean;
+  /**
+   * Tooltip to show on the body cell
+   * Note: If the body cell is truncated and has simple string content, it will already attempt to display the cell text
+   * If you want to show a tooltip that differs from the cell text, you can set it here
+   * To disable it completely you can set it to null
+   */
```

Comment:
This block could use some punctuation

<img width="878" alt="Screen Shot 2022-10-03 at 11 09 31 AM" src="https://user-images.githubusercontent.com/35148959/193626385-f3549b6a-8701-4d76-aecc-8d19680f4b22.png">

Reply:
I'll update that (and the existing `Th` one I copied it from)